react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1133,18 +1134,18 @@ const SessionLauncherPage = () => {
                           <>
                             <Descriptions.Item
                               label={t('session.launcher.StartUpCommand')}
+                              contentStyle={{
+                                overflow: 'auto',
+                              }}
```

Comment:
First, I solved it with line wrapping instead of scrolling but I have a small question.

Is it possible to control the scrolling within the Source Code Viewer itself without setting a specific width for Descriptions.Item? From a Description's perspective, shouldn't the overflow of content be managed by Description itself? @yomybaby

Reply:
Sorry, last time I was confused about some parts of the code.